test/test.js

```diff
@@ -315,4 +316,40 @@ describe('createDiff', function() {
             });
         });
     });
+
+    it('should return a buffer if save option is set to false', function(done) {
+        looksSame.createDiff({
+            reference: srcPath('ref.png'),
+            current: srcPath('different.png'),
+            diff: this.tempName,
+            highlightColor: '#ff00ff',
+            save: false
+        }, function(error, buffer) {
+            expect(buffer).to.be.an.instanceof(Buffer);
+            done();
+        });
+    });
+
+    it('should return a buffer equal to the diff on disk', function(done) {
+        var _this = this;
+        looksSame.createDiff({
+            reference: srcPath('ref.png'),
+            current: srcPath('different.png'),
+            diff: this.tempName,
+            highlightColor: '#ff00ff',
+        }, function(error) {
+            looksSame.createDiff({
```

Comment:
I see the author uses some [images](https://github.com/flore77/looks-same/tree/125c432f5f58e65eb9d68540c952dc596ba132c4/test/data/src) for these tests. You should read the diff from there instead of calling `createDiff`. I'm assuming [this](https://github.com/flore77/looks-same/blob/125c432f5f58e65eb9d68540c952dc596ba132c4/test/data/diffs/small-magenta.png) is the right image.